constructor/osx/run_installation.sh

```diff
@@ -24,12 +27,11 @@ CONDA_EXEC="$PREFIX/conda.exe"
 
 # Perform the conda install
 notify "Installing packages. This might take a few minutes."
-CONDA_SAFETY_CHECKS=disabled \
+if ! CONDA_SAFETY_CHECKS=disabled \
```

Comment:
I am not sure this way of expressing the same thing is more readable or not. The other one is clearer in my opinion (checking the "booleanness" of the return code).

Reply:
see https://www.shellcheck.net/wiki/SC2181

The previous code had to be changed due to `set -e` being enabled.